src/widget/wtrackmenu.cpp

```diff
@@ -1725,10 +1733,19 @@ void WTrackMenu::slotRemoveFromDisk() {
         delListWidget->addItems(locations);
         mixxx::widgethelper::growListWidget(*delListWidget, *this);
         // Warning text
+        QString delWarningText;
+        if (m_pTrackModel) {
+            delWarningText = tr("Permanently delete these files from disk?") +
+                    QString("<br><br><b>") +
+                    tr("This can not be undone!") + QString("</b>");
+        } else {
+            delWarningText =
```

Comment:
On Linux you can delete the file while playing.
It is actually removed after the last reference to the file has been closed.
On Windows the file is locked until all file handles are closed. 
I am not sure how it relates to caching. The file is kept in cache after rejecting. 
We need to check if the lock is released than.